tests/llama_index/conftest.py

```diff
@@ -86,11 +88,13 @@ def document():
 
 
 @pytest.fixture
+@trace_disabled
```

Comment:
note: Creating a vector index produces a trace (cuz it actually runs embeddings, etc), we suppress it so it won't be mixed by the prediction trace.